### PR TITLE
.NET: simplify initialization snippets

### PR DESCRIPTION
**Pull request by Tyrrrz**
@bruno-garcia 
Realted to #3407

### CODE REVIEW

File: src/includes/configuration/sample-rate/dotnet.mdx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ```csharp
 using Sentry;
 
+// Add this to the SDK initialization callback
 using (SentrySdk.Init(o =>
```
Comment (bruno-garcia) [inline src/includes/configuration/sample-rate/dotnet.mdx:5]: Still doing init

Comment (Tyrrrz) [reply src/includes/configuration/sample-rate/dotnet.mdx:5]: oops

Comment (Tyrrrz) [reply src/includes/configuration/sample-rate/dotnet.mdx:5]: here we have `// app code here` so i just left it as it is.
removed my comment.

**Review by bruno-garcia — COMMENTED** [review]
Everything here makes sense but browsing the code I continue to see SentrySdk.Init such as:

```
using (SentrySdk.Init(o =>
{
    o.SampleRate = 0.25; // Capture 25%
})
{
    // app code here
}
```

**Comment by bruno-garcia** [comment]
under `configuration/sampling/`

**Comment by bruno-garcia** [comment]
I went through all docs and only spotted that one.

**Review by bruno-garcia — APPROVED** [review]

**Pull request merged**